SR2020/2020-02-19-inactive-team-prompt.md

```diff
@@ -0,0 +1,49 @@
+---
+to: Teams deemed 'inactive'
+subject: Student Robotics inactivity
```

Comment:
This subject line is pretty blunt. This would make me less likely to become active.